### DISCUSSION TITLE
[Feature Request] - OpenAi integration

### DISCUSSION THREAD

**Discussion by luciano2000** [GetPublii/Publii#1573] [Ideas]
We know that today content creation (or assistance) via OpenAI is present in the best CMSs through plugins. Some can even read external RSS feeds and, through the correct prompts, modify the text for us to publish.

Is there any project or plan for integrating OpenAI with Publii?

**Comment by johnwoltman** [comment]
I'm on the AI bandwagon but there's already dozens of services that can help generate AI slop articles.

**Reply by luciano2000** [reply]
> I'm on the AI bandwagon but there's already dozens of services that can help generate AI slop articles.

Automatically? I don't know of any tool integrated into Publii for this.

**Reply by johnwoltman** [reply]
Oh, not for Publii.  I meant in general.  Half the web seems like AI slop now.  I can't imagine needing Publii to help make more.

**Reply by candidexmedia** [reply]
I agree @johnwoltman. The concept of using an OpenAI plugin to "read external RSS feeds and, through the correct prompts, modify the text for us to publish" sounds like borderline theft/plagiarism, but I hope I'm interpreting this incorrectly. In any case, there's nothing stopping someone from just generating the content on chat gpt and pasting it into Publii (or even straight up stealing someone else's content).

> I can't imagine needing Publii to help make more.

Plus, I can't imagine that having Publii associated with low-quality AI content mills to be a good look for both the devs and users.

I'm not 100% against the idea, but I'd much rather have AI in Publii that supports grunt work, rather than replacing the part of blogging that many find enjoyable — the actual writing. Additionnally, a local, actually open-source model would be preferable.

I'd support something like:
- A generator for meta descriptions based on your content, and seo optimized
- A bot that scan for images that are missing alt text, and makes suggestions based on the image's content
- Anything that helps Publii userd improve their accessibillity
- A chatbot/search engine plugin which site visitors can query to get summaries of info
- Enhancing the related posts function ([see example from Ghost](https://forum.ghost.org/t/ghostai-ghost-openai-integration/37578))
